packages/appeals-service-api/src/mappers/questionnaire-submission/has-mapper.js

```diff
@@ -0,0 +1,87 @@
+class HasQuestionnaireMapper {
+	mapToPINSDataModel(journeyResponse) {
+		return [
+			{
+				questionnaire: {
+					LPACode: journeyResponse.LPACode,
+					caseReference: journeyResponse.caseReference,
+					isAppealTypeAppropriate: journeyResponse.answers['correct-appeal-type'] ? true : false,
+					doesTheDevelopmentAffectTheSettingOfAListedBuilding: journeyResponse.answers[
+						'affects-listed-building'
+					]
+						? true
+						: false,
+					affectedListedBuildings: () => {
+						if (journeyResponse.answers['affects-listed-building']) {
+							return journeyResponse.answers['listed-building-number'];
+						}
+					},
+					siteWithinGreenBelt: journeyResponse.answers['green-belt'] ? true : false,
+					howYouNotifiedPeople: journeyResponse.answers['notification-method'] ? true : false,
+					doesSiteHaveHealthAndSafetyIssues: journeyResponse.answers['safety-risks'] ? true : false,
+					healthAndSafetyIssuesDetails: () => {
+						if (journeyResponse.answers['safety-risks']) {
+							return journeyResponse.answers['affects-listed-building'];
+						}
+					},
+					doesSiteRequireInspectorAccess: journeyResponse.answers['inspector-visit-appellant']
+						? true
+						: false,
+					//journeyResponse.answers['inspector-visit-neighbour'] - should this be housed in the same property as above possibly doNeightboursAffectNeighboringSite
+					// newPlanningConditions: journeyResponse.answers['new-planning-conditions'] ? true : false, // fieldname needs clarifying
+					// journeyResponse.answers['other-ongoing-appeals'] // fieldname needs clarifying
```

Comment:
the fieldname for the boolean is `new-planning-conditions`
the fieldname for the text value if the user answers 'yes' for the above is `new-planning-conditions_new-conditions-value`